pkg/network/setup/netpod/netpod_test.go

```diff
@@ -940,15 +941,6 @@ var _ = Describe("netpod", func() {
 			netpod.WithCacheCreator(&baseCacheCreator),
 		)
 		Expect(netPod.Setup()).To(Succeed())
-		Expect(nmstatestub.spec).To(Equal(
-			nmstate.Spec{
-				Interfaces: []nmstate.Interface{},
-				LinuxStack: nmstate.LinuxStack{IPv4: nmstate.LinuxStackIP4{
-					PingGroupRange:        []int{107, 107},
-					UnprivilegedPortStart: pointer.P(0),
-				}},
-			},
-		))
```

Comment:
Why this was removed? The production code has not changed except adding a comment.

Reply:
I meant to remove the relevant production code but pushed the wrong diff, fixed

Reply:
Fixed, the relevant production code is removed, following this thread https://github.com/kubevirt/kubevirt/pull/11915/commits/e81b8744089ef40da9c3d8a2249beaa541c20e50#r1611582617

Reply:
I think you should not touch the reconcile loop.
Have we discussed changing the reconcile of the virt-handler anywhere and concluded that it can be changed?

Reply:
Back in macvtap deprecation I said we can remove the leftovers at the network setup code because it run once and cause no disruption to running VMs.
There was agreement to leave the remaining code to be on the safe side.

From what I have tested with passt we can remove the remaining, from now I am reverting this change.
Hopefully we can get rid of the remaining in a follow up PR.
 
Done.